src/open_inwoner/templates/pages/cases/status_inner.html

```diff
@@ -33,12 +33,59 @@ <h2 class="h2" id="statuses">{% trans 'Status' %}</h2>
                 <aside class="status-list" aria-label="{% trans "Status lijst" %}">
                     <ul class="status-list__list">
                         {% for status in case.statuses %}
-                        <li class="status-list__list-item">
-                            {% icon icon="task_alt" %}
-                            <p class="p p--compact">{{ status.date|date }}</p>
-                            <p class="p p--compact">{{ status.label }}</p>
-                        </li>
+
+                            {% if forloop.last %}
+                                {# Current active status, including final status #}
+                                <li class="{% if case.end_statustype_data %}status--current{% endif %} status-list__list-item status-list__list-item--{{ status.status_indicator }} {% if not case.end_statustype_data %}item--completed status-list__list-item--info{% endif %}">
+                                    {% icon icon="circle" %}
+                                    <div class="status-list__notification status-list__notification--{{ status.status_indicator }}">
+                                        <h3 class="status-list__notification-h3"><a href="#" class="link link-toggle link--bold">{{ status.label }}</a>{% icon icon="expand_more" icon_position="after" outlined=True %}</h3>
+                                        <div class="status-list__notification-content">
+                                            <p class="p p-text p--compact">{{ status.status_indicator_text }}</p>
+                                            <p class="p p-date p--compact">{{ status.date|date }}</p>
+                                            <p>
+                                            {% if case.internal_upload_enabled or case.external_upload_enabled %}
```

Comment:
This works correctly for status that are currently active, for cases that are not finished yet,
but the button doesn't show up in the last status for Cases that are finalized ('Afgehandeld').
Example: for ISBN login nr 111111110: http://127.0.0.1:8000/mijn-aanvragen/5a7520bb-789d-401f-bdc5-e3a2e92cfeee/status/
Maybe it helps to say I am using this kind of code. in order to show things specifically for the last finalized status only:
`{% if not case.end_statustype_data %}LAST{% endif %}`